### PR TITLE
travis: test against go 1.[78].x

### PR DESCRIPTION
**Pull request by dajohi**

### CODE REVIEW

File: zkclient/net.go [hunk 1]
```diff
@@ -40,18 +40,8 @@ func (mw *mainWindow) kx(conn io.ReadWriteCloser) error {
...
-	if err != nil {
-		return err
-	}
 
-	// handle trust
-	//err = handleTrust(remoteId)
-	//if err != nil {
-	//	// printing has happened
-	//	return err
-	//}
```
Comment (jrick) [inline zkclient/net.go]: um, did gosimple remove this or did you?  seems like commented code should be left.

Comment (dajohi) [inline zkclient/net.go]: i removed it.. the function doesn't exist.  perhaps i should leave a single line
```// TODO handle trust```  ?

or make an issue on GH?

Comment (marcopeereboom) [inline zkclient/net.go]: Man just leave that alone.

Comment (dajohi) [inline zkclient/net.go]: Left it in.

**Pull request merged**